src/Cedar/Interop_OpenVPN.h

```diff
@@ -312,7 +312,7 @@ struct OPENVPN_SERVER_UDP
 };
 
 // OpenVPN Default Client Option String
-#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-128-CBC,auth SHA1,keysize 128,key-method 2,tls-client"
+#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-256-CBC,auth RMD160,keysize 128,key-method 2,tls-client"

```

Comment:
Why are AES_128 and SHA1 been removed? they are currently used in running setups, and this change will probably break them. Please add not replace. Unless the algorithm is really broken from a security POV